### ISSUE TITLE
Why not resnet?

### ISSUE THREAD

**Issue by yangbisheng2009** [GantMan/nsfw_model#23]
Since resnet is very strong now, why didn't your try it?
Is it something wrong or some theory not suitable for nsfw？

Looking forward to your guidance

**Comment by GantMan** [comment]
Good question!   To be fair, when I started this model, I was shooting for the best.

I think resnet is a great suggestion.   I'll see about training a resnet version next!

**Comment by TechnikEmpire** [comment]
In 1.2.1, an upcoming release, there may be Resnet 50 V2 models that I've trained. However, Efficientnet models are also trained that are superior, they just can't be deployed to JS yet.

**Comment by BerryRB** [comment]
@TechnikEmpire  Hi, can you tell me when will Version 1.2.1 be released?

**Comment by TechnikEmpire** [comment]
@BerryRB just look at the open PR I have for it. There are linked models there. You wont want resnet though its huge and offers little advantage over other models. In my fork I've made a readme page where you can see the stats of all trained models I've linked in the PR. @GantMan is the repo owner and its up to him to merge and officially post models but last time we spoke, he said he had a crazy rammed schedule so it might  be a while before an official release.

**Comment by yangbisheng2009** [comment]
https://github.com/yangbisheng2009/nsfw-resnet

I implement NSFW by pytorch-resnet101
Highly thanks GantMan，give me excellent suggestion

**Comment by BerryRB** [comment]
@TechnikEmpire  Can you tell me the amount of your training data and test data? Can you share your test data with me if you can? I currently have 2W cleaned data, which I can share with you if you need
